files/00_setup_freeipa.sh

```diff
@@ -155,6 +155,14 @@ function setup {
     ipa host-add-principal \
         "$hostname" \
         host/"$(curl --silent http://169.254.169.254/latest/meta-data/instance-id)"."$domain"
+
+    # Enable features in the active authselect profile so that all necessary
+    # hardened rules will be activated.  Note that these features are enabled
+    # in ansible-role-hardening, however ipa-server-install and
+    # ipa-client-install clobber them, so we must re-enable them here.
```

Comment:
I suggest adding to the comment a sentence addressing the fact that these commands are RedHat-only, but so are IPA servers.

Reply:
Thanks - see updated comment in https://github.com/cisagov/ansible-role-freeipa-server/pull/23/commits/1b0fb61563810eee0a3ce3bed5c12c75ca95aa2e.